staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -42,11 +43,18 @@ import (
 type AdmissionOptions struct {
 	// RecommendedPluginOrder holds an ordered list of plugin names we recommend to use by default
 	RecommendedPluginOrder []string
-	// DefaultOffPlugins a list of plugin names that should be disabled by default
-	DefaultOffPlugins []string
-	PluginNames       []string
-	ConfigFile        string
-
+	// DefaultEnabledPlugins is a set of plugin names that is enabled by default
+	DefaultEnabledPlugins sets.String
```

Comment:
I don't think this is necessary. If you maintain the order and you track the `DefaultDisable` set, then the remainder are default enabled.

Reply:
ok